great_expectations/core/util.py

```diff
@@ -782,148 +769,124 @@ def sniff_s3_compression(s3_url: S3Url) -> Union[str, None]:
     return _SUFFIX_TO_PD_KWARG.get(s3_url.suffix) if s3_url.suffix else None
 
 
-# noinspection PyPep8Naming
-def get_or_create_spark_application(
```

Comment:
@NathanFarmer 
Is the utility not user facing? Wouldn't it be a breaking change?

I would suggest keeping the function but having it raise a deprecation warning and call the renamed method.
Although I'm not sure if the usage actually makes sense but you would know.
```python
def get_or_create_spark_application():
   warnings.warn("Don't do it", category=DeprecationWarning)
   return get_or_create_spark_session()
```